package.json

```diff
@@ -155,7 +155,8 @@
     "postinstall": "node ./scripts/postinstall.js",
     "test": "node ./out/src/test/runTest.js",
     "tslint": "gulp tslint",
-    "ui-test": "npm run compile && extest setup-and-run './out/src/ui-test/*_test.js'"
+    "preui-test": "npm run compile && rimraf test-extensions && mkdir test-extensions",
```

Comment:
Where is this script used?

Reply:
It is launched before `ui-test` script automatically by npm.